core/src/message/init.rs

```diff
@@ -23,7 +23,9 @@ use crate::message::{
 use codec::{Decode, Encode};
 use scale_info::TypeInfo;
 
-/// Init message.
+/// InitMessage
```

Comment:
No need to repeat the name of the structure.
```suggestion
```

Reply:
Removed this line